src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous" >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__container">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Product">
+
+        <h1 class="card__name">APPLE A1419 iMac 27" Retina  5K Monoblock (MNED2UA/A)</h1>
```

Comment:
Code format again :)

Also don't use `h1` here - generally, it is advised to have a single h1 per page. You are designing the product card - it is obvious the store will have more than a single product on the page.

I would suggest to have `p` here or at least some "lower grade" `h`
See good article on `hx` tags: https://semji.com/blog/why-use-h1-h2-h3-tags-for-your-seo